scripts/zones/Aydeewa_Subterrane/IDs.lua

```diff
@@ -20,6 +20,8 @@ zones[xi.zone.AYDEEWA_SUBTERRANE] =
         MEMBERS_LEVELS_ARE_RESTRICTED = 7023, -- Your party is unable to participate because certain members' levels are restricted.
         FISHING_MESSAGE_OFFSET        = 7057, -- You can't fish here.
         MINING_IS_POSSIBLE_HERE       = 7328, -- Mining is possible here if you have <item>.
+        PW_WHO_DARES                  = 7964, -- Pandemonium Warden: Who dares disturb these gates...
+        PW_END_OF_NOTHING             = 7965, -- Pandemonium Warden: This is the end...of nothing...
```

Comment:
The comments following text IDs need to match exactly what we have listed in our extracts in UpdateExtractor: https://github.com/LandSandBoat/UpdateExtractor/blob/main/out/scripts/zones/Aydeewa_Subterrane/Text.lua.raw.txt#L7870-L7871

Otherwise, when/if there are ID shifts during version updates our automatic tooling will fail

Reply:
Wow good info, thanks!